### DISCUSSION TITLE
Executing multiple SQL files.

### DISCUSSION THREAD

**Discussion by jensenkd** [Snowflake-Labs/snowcli#1808] [Q&A] [answered]
When using Snow cli and using the SQL command, is there a way to execute multiple files? Or have an entry file that references multiple SQL files?

It looks like SnowSQL has this capability, wondering if there is equivalent approach?

**Comment by sfc-gh-turbaszek** [comment, marked as answer]
Yes, you can execute `snow sql -f my.sql -f other.sql -f more.sql` see:
